cl/audio/migrations/0010_switch_to_s3_storage_noop.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.1.7 on 2021-04-07 00:24
+
+from django.db import migrations, models
+
+import cl.lib.model_helpers
+import cl.lib.storage
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('audio', '0009_abstract_datetime_model'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='audio',
+            name='local_path_mp3',
+            field=models.FileField(blank=True, db_index=True, help_text='The location, relative to MEDIA_ROOT, on the CourtListener server, where encoded file is stored', storage=cl.lib.storage.AWSMediaStorage(), upload_to=cl.lib.model_helpers.make_upload_path),
+        ),
+        migrations.AlterField(
+            model_name='audio',
+            name='local_path_original_file',
```

Comment:
Future comment, I think we should either alter the help text or update the fields to reflect the audio being on S3 and not the local courtlistener server?  We refer to files in LASC as s3_filepath.

Reply:
I decided against updating the fields themselves at least for now, because that'd break the API, but you're right, the help text needs a rework. Good one.